utilities/ismrmrd_stream_to_hdf5.cpp

```diff
@@ -88,6 +79,11 @@ void convert_stream_to_hdf5(std::string output_file, std::string groupname, std:
             throw std::runtime_error(ss.str());
         }
     }
+
+    // If we can read any more at this point, it is an error
+    if (is.rdbuf()->in_avail() > 0) {
```

Comment:
I could be wrong, but isn't this a function of the internal buffering state, not necessarily the true end of stream?